sirepo/package_data/nersc_test/sequential_test.sh.jinja

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -eou pipefail
+cd sirepo_run_dir
+export SIREPO_MPI_CORES=1
+export PYKERN_PKCONFIG_CHANNEL=dev
+export PYKERN_PKDEBUG_WANT_PID_TIME=1
+export PYTHONUNBUFFERED=1
+export SIREPO_AUTH_LOGGED_IN_USER={{user}}
+export SIREPO_SIMULATION_DB_LOGGED_IN_USER={{user}}
+export PYTHONPATH=
+export PYTHONSTARTUP=
+perl -p -e "s<{{sirepo_run_dir}}><$PWD>" '{{json_in_path}}' > 'jobCmdIn.json'
```

Comment:
This is not correct, because there is no `{{sirepo_run_dir}}` in the json file. Indeed, you should simply generate jobCmdIn.json (or just call it sequential_test.json) in nersc_test.py.